cypress-io/cypress
# fix: test-recording instructions in Component Test mode

- Closes https://cypress-io.atlassian.net/browse/UNIFY-1637

### User facing changelog

In component testing mode, when no runs have been recorded to the dashboard, the command to copy will now include the `--component` flag, so that we run in component testing mode.

Additionally, the project root has been removed from the terminal prompt in all situations (per Brian's comment in this ticket). Terminal prompts now begin with `$` always, instead of `some-project: $`.

### How has the user experience changed?

Before - a command is shown in component testing mode, but this command would run cypress in the default e2e mode, thus not record component tests:
![Screen Shot 2022-05-10 at 6 01 56 PM](https://user-images.githubusercontent.com/8340719/167729154-cac87fe8-c48d-4e00-9257-925ecdaa93b7.png)

After - correct command is shown for component testing (it includes the `--component` flag:
![Screen Shot 2022-05-10 at 5 59 46 PM](https://user-images.githubusercontent.com/8340719/167729265-9628df8d-8de3-4843-b3af-cb7fa9b76a69.png)

### How to test manually

Set up a project with no tests recorded in the dashboard and visit the runs page in the app, it should invite you to record your first run, and recommend the correct command for your testing type.

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [na] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?
